src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -298,10 +313,12 @@ def parse_config(job, shared_ids, input_args):
 
     with open(config, 'r') as f_in:
         for line in f_in:
-            line = line.strip().split(',')
-            uuid = line[0]
-            urls = line[1:]
-            samples.append((uuid, urls))
+            if line.strip():
+                line = line.strip().split(',')
+                assert len(line) == 3, 'Improper formatting'
```

Comment:
@hannes-ucsc and I argued a bit about this a while ago, but I prefer raising named exceptions when validating user input (e.g., if a user provides bad arguments, throw a `IllegalArgumentException` as a Java equivalent) and asserts for checking program invariants (i.e., if the assert failed, there is a bug in the code somewhere, not bad user input).

Also, I'd prefer a more detailed error message, e.g., `Improper formatting. Expected 3 fields, received...`